### PR TITLE
fix(Resources status/Downtime): Incorrect default downtime duration

### PR DESCRIPTION
**Pull request by Thebarda**
## Description

This fixes an incorrect default downtime duration usage. Instead of to use hard coded default downtime duration, we use the one defined in `Administration > Parameters > Monitoring`

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 19.10.x
- [ ] 20.04.x
- [x] 20.10.x
- [x] 21.04.x (master)

<h2> How this pull request can be tested ? </h2>

- Define a custom default downtime duration in `Administration > Parameters > Monitoring` (Example: 120 seconds)
- Come back to Resources status page
- Click on downtime icon on a resource
 --> The default duration should be the one that you setted up before
 --> The end date should be the current date + default duration (in seconds)

## Checklist

- [x] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

**Comment by UrBnW** [comment]
I think that this should fix #8705.
Thank you 👍

File: www/front_src/src/Provider/models.ts [hunk 1]
```diff
@@ -3,7 +3,8 @@ import { KeyValuePair } from 'ramda';
 export interface User {
   locale: string;
   timezone: string;
-  username: string;
+  name: string;
+  alias: string;
 }
 
 export type UserContext = {
```
Comment (bdauria) [inline www/front_src/src/Provider/models.ts:10]: This should be moved to the ui-context repo, along with the other context-related types

File: www/front_src/src/Resources/Actions/Resource/Acknowledge/index.tsx [hunk 2]
```diff
@@ -36,7 +36,7 @@ const AcknowledgeForm = ({
   const { t } = useTranslation();
   const { showMessage } = useSnackbar();
 
-  const { username } = useUserContext();
+  const { name } = useUserContext();
```
Comment (bdauria) [inline www/front_src/src/Resources/Actions/Resource/Acknowledge/index.tsx:39]: We should use alias instead of name here.

**Review by bdauria — APPROVED** [review]
LGTM & tested. We might need to handle the parameters edition better because right now we have to logout / login again to update them -> but that's something for the backend team

**Review by jdelpierre — APPROVED** [review]

**Pull request merged**